src/helpers/get-leading-account.ts

```diff
@@ -28,11 +28,12 @@ export function getLeadingAccount() {
   while (!leadingAccount && index < walletTypes.length) {
     const walletType = walletTypes[index];
 
-    const w = wallets
-      .filtered(`type = '${walletType}' and isHidden = false`)
-      .sorted('path');
-    if (w.length) {
-      leadingAccount = w[0].address;
+    const wallet = wallets
+      .filter(w => w.type === walletType)
```

Comment:
Add a condition to the filter: `isHidden = false`

Reply:
Added